web.primefaces/licenseinfo.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/web/primefaces/resources/welcomePrimefaces.template</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/jsfcrud.css</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/jsfcrud.js</file> 
```

Comment:
Should we add the license header directly to jsfcrud.js and jsfcrud.css rather than adding them to licenseinfo.xml?

Reply:
The templates and snippet files below are freemarker templates and the license is inside template comment markers, so don't end up in user code. You could turn `jsfcrud.*` into freemarker templates - I would support that approach, but I think having them here is in line with current handling in netbeans.